server/stream.go

```diff
@@ -318,6 +365,9 @@ func checkStreamCfg(config *StreamConfig) (StreamConfig, error) {
 	if cfg.MaxConsumers == 0 {
 		cfg.MaxConsumers = -1
 	}
+	if cfg.Duplicates == 0 {
```

Comment:
this is allowing negative times, is that intentional? For documentation whats the intended behaviour of negative times? I think we should fail to create the stream in that case if not intentional

Reply:
Also should we set a upper limit maybe? It would be bad if someone set this to a year

Reply:
```
$ nats req -H Msg-Id:1 in.x hello
11:55:24 Sending request on [in.x]
11:55:24 Received on [_INBOX.YyMIz3ytiJtzdus9c8kbXj]
+OK {"stream": "X", "seq": 1}
```

What do you think about adding a field to the response in the case of a duplicate ie `+OK {"stream": "X", "seq": 1, "dupe": true}` or similar?

Reply:
These are good suggestions. Will add the test for negative and also add in `duplicate` field. What feels like a good upper bound? I agree but do not know what the right number is TBH. For now I may suggest we keep open, we can add one later.

I can limit if they have a limit on the stream itself as well.

Reply:
difficult to say what a good upper limit is tbh, but surely a day should be plenty, hard to be sure what the use cases for this is tbh